odoo/addons/base/views/res_partner_views.xml

```diff
@@ -150,8 +150,9 @@
                 </div>
                 <sheet>
                     <div class="oe_button_box" name="button_box"/>
+                    <field name="avatar_128" invisible="1"/>
```

Comment:
no need to add it if it wasn't there?
It's only useful to avoid http request for images, at the cost of a much bigger read request.